src/test/java/com/epam/tc/hw1/MultipleCalculatorTest.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class MultipleCalculatorTest extends TestingCalculatorOperations {
+
+    @DataProvider(name = "DBMultiple")

```

Comment:
What does DB stand for? Data Base?

Reply:
That's a typo, thanks for pointing it out. It was meant to be a DP, as in other similar classes, I'll fix it.